package.json

```diff
@@ -146,7 +146,7 @@
     "puppeteer": "^1.3.0",
     "qunit": "^2.5.0",
     "route-recognizer": "^0.3.4",
-    "router_js": "^6.1.3",
+    "router_js": "^6.1.4",
```

Comment:
Hmm, this is a bit odd, seems like we are getting 6.2.1 instead of 6.1.4. Would you mind pinning this minimum version to the current latest release?

Reply:
6.2.0 has the `buildRouteInfoMetadata` stuff in it.

Reply:
But as you can see from the `yarn.lock` we are _already_ getting 6.2.1 in the `yarn.lock` so we might as well "just do it"...